bundle/libraries/libraries.go

```diff
@@ -0,0 +1,43 @@
+package libraries
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/log"
+)
+
+type attach struct {
+}
+
+func AttachToResources() bundle.Mutator {
+	return &attach{}
+}
+
+func (a *attach) Name() string {
+	return "libraries.AttachToResources"
+}
+
+func (a *attach) Apply(ctx context.Context, b *bundle.Bundle) error {
+	r := b.Config.Resources
+	for k := range b.Config.Resources.Jobs {
+		tasks := r.Jobs[k].JobSettings.Tasks
+		for i := range tasks {
+			task := &tasks[i]
+			if task.PythonWheelTask != nil {
+				packageName := task.PythonWheelTask.PackageName
+				artifact, ok := b.Config.Artifacts[packageName]
```

Comment:
We assume here that the package name is identical to the key in the artifacts map. How can we be sure? If the generated package name from setup.py is different, the job won't run.

Reply:
@pietern yes, this is an assumption we agreed on during design reviews. If users make a choice to use different artifact name they will have to include it in `Libraries` section explicitly

Reply:
We'd have to take care to autodetect this name if not specified in the artifacts block.

Reply:
👍